### PR TITLE
adding punjabi language

### LINKED ISSUE #9405: Activate pa-IN Language
Website is [localized](https://github.com/mozilla/addons-server/tree/master/locale/pa_IN) in punjabi language but need to activate on server side.

### LINKED ISSUE #9406: activating pa-IN language
Fixes #9405

### PR DESCRIPTION
**Pull request by championshuttler**
Fixes #9405

### CODE REVIEW

File: src/olympia/lib/settings_base.py [hunk 1]
```diff
@@ -202,6 +202,7 @@ def get_db_config(environ_var):
     'nb-NO',  # Norwegian (Bokmål)
     'nl',  # Dutch
     'nn-NO',  # Norwegian (Nynorsk)
+    'pa-IN', # Punjabi
```
Comment (eviljeff) [inline src/olympia/lib/settings_base.py]: see codestyle fail

File: src/olympia/constants/search.py [hunk 2]
```diff
@@ -81,6 +82,7 @@
     'hungarian': 'light_hungarian',
     'indonesian': 'indonesian',
     'italian': 'light_italian',
+    'punjabi': 'punjabi',
```
Comment (eviljeff) [inline src/olympia/constants/search.py]: Unfortunately ElasticSearch doesn't seem to support a specific stemmer for Punjabi https://www.elastic.co/guide/en/elasticsearch/reference/5.3/analysis-stemmer-tokenfilter.html so you'll have to omit this line.

File: src/olympia/lib/settings_base.py (hunk 1 above)
Comment (championshuttler) [reply src/olympia/lib/settings_base.py]: Sorry got it , made the changes

**Comment by EnTeQuAk** [comment]
Restarted the failing ES job, looked like a Travis failure.

File: src/olympia/constants/search.py [hunk 3]
```diff
@@ -22,6 +22,7 @@
     'german': ['de'],
     'greek': ['el'],
     'english': ['en-us', 'en-ca', 'en-gb'],
+    'punjabi': ['pa-IN'],
```
Comment (diox) [inline src/olympia/constants/search.py]: Don't do this. Elasticsearch does not have a `punjabi` analyzer.

Comment (championshuttler) [reply src/olympia/constants/search.py]: Thanks, removed it , lets see how does travis play now :)

**Comment by championshuttler** [comment]
@diox thanks for it , finally travis test cases passed :star:

**Pull request merged**